bbb-graphql-server/bbb_schema.sql

```diff
@@ -2486,8 +2497,13 @@ JOIN "pluginDataChannelEntry" m ON m."meetingId" = u."meetingId"
 				OR u."role" = ANY(m."toRoles")
 				OR (u."presenter" AND 'PRESENTER' = ANY(m."toRoles"))
 				)
-WHERE "deletedAt" is null
-ORDER BY m."createdAt";
+WHERE "deletedAt" is null AND "isPublic" = false;
+
+CREATE OR REPLACE VIEW "v_pluginDataChannelEntry_public" AS
+SELECT u."meetingId", m."pluginName", m."channelName", m."subChannelName", m."entryId", m."payloadJson", m."createdBy", m."toRoles", m."createdAt", m."updatedAt"
+FROM "user" u
```

Comment:
You can query directly from `pluginDataChannelEntry`.
The current way the result will multiply by the number of the users in the room.